text-templates/repo-aws-iam-policy.json

```diff
@@ -0,0 +1,50 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "ListPublicData",
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket"
+            ],
+            "Resource": [
+                "arn:aws:s3:::nextstrain-data",
+                "arn:aws:s3:::nextstrain-staging"
+            ],
+            "Condition": {
+                "StringLike": {
+                    "token.actions.githubusercontent.com:aud": "sts.amazonaws.com",
+                    "token.actions.githubusercontent.com:sub": "repo:${FULL_REPO_NAME}:*:job_workflow_ref:nextstrain/.github/.github/workflows/pathogen-repo-build.yaml@*",
```

Comment:
We shouldn't need the conditions on the OIDC token here (or in the next policy statement below). They're already enforced by the role trust policy and by the time we've successfully assumed the role and have a valid session that this session policy applies to, we're good.

Reply:
Huh, okay. I wasn't entirely sure how these would "merge" for the role policy and session policy.

Reply:
Hmm, good question.

Generally, I'm now wondering how much "session policy intersected with role policy" is going to be painful for us to reason about. It seemed straightforward when we last discussed, but I'm feeling like it's less so now.

Reply:
No longer relevant with [new approach](https://github.com/nextstrain/.github/compare/b2c38a04e15c817dba16b360eefbd2d89ecaaf11..58ba6a0afd97a30bfcf9fee85dbe48e7c9526b40).